docs/uplink/installation.md

```diff
@@ -0,0 +1,93 @@
+# Install inlets-uplink
+
+Inlets Uplink is a self-hosted solution for managing many tunnels for a team or company.
+
+## Pre-reqs
+* A Kubernetes cluster with LoadBalancer capabilities (i.e. public cloud).
+* A domain name clients can use to connect to the tunnel control plane.
+* [arkade](https://github.com/alexellis/arkade), a simple CLI tool that provides a quick way to install various apps and download common binaries much quicker
+    
+    To install arkade run:
+
+    ```bash
+    curl -sSLf https://get.arkade.dev/ | sudo sh
+    ```
+
+## Create a Kubernetes cluster
+
+Create a cluster with at least 3 nodes 2-4GB of RAM each
+
+## Install the IngressController
+
+This installs nginx-ingress using its Helm chart:
+
+```bash
+arkade install ingress-nginx
+```
+
+## Install cert-manager
+
+Install [cert-manager](https://cert-manager.io/docs/), which can be integrated with NginxIngress to manage TLS certificates.
+```bash
+arkade install cert-manager
+```
+
+## Deploy inlets-uplink with Helm
+Make sure to create the target namespace for you installation first.
+
+```bash
+kubectl create namespace inlets
+```
+
+Create the required secret with your inlets-uplink license.
+
+```bash
+kubectl create secret generic \
+  -n inlets inlets-uplink-license \
+  --from-file license=$HOME/.inlets/LICENSE_UPLINK
+```
+
+Now deploy inlets-uplink from the OCI Helm chart repo:
+
+```bash
+# Domain used to expose the control-plane.
+# Clients will connect with a URI of:
+# wss://uplink.example.com/namespace/tunnel
+export DOMAIN="uplink.example.com"
+# Email address used for ACME registration
+export EMAIL="user@example.com"
+
+helm upgrade --install inlets-uplink \
```

Comment:
I think we should show a values.yaml approach instead

This will be more conducive for people who want to install with a GitOps tool, and is easier for us to edit.

You can still cat >> EOF if you like to generate the values.yaml